### DISCUSSION TITLE
The "share feedback" link for lists links to the wrong place

### DISCUSSION THREAD

**Discussion by henrebotha** [community/community#28305] [Other Feature Feedback, Questions, & Ideas]
Clicking the link in the screenshot below takes me to https://github.com/community/community/discussions/categories/lists-feedback, instead of the proper https://github.com/community/community/discussions/categories/lists.

![Feedback link goes to the wrong place](https://user-images.githubusercontent.com/5593874/182816376-17b1fd9c-4cab-4130-96de-40351bdd6efe.png)

Anyway, only discovered this because I wanted to give some feedback, which is that the feature is sick as hell and I use it a lot.

**Comment by jfullstackdev** [comment]
yes, I can reproduce the same thing, they need to update the link
